bot/internal/bot/docpaths.go

```diff
@@ -3,13 +3,12 @@ package bot
 import (
 	"context"
 	"encoding/json"
-	"errors"
-	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/gravitational/shared-workflows/bot/internal/github"
 	"github.com/gravitational/trace"
+	"github.com/spf13/afero"
```

Comment:
Can we use `io/fs` instead of pulling in a dependency?

Reply:
Done in [9c4ac80](https://github.com/gravitational/shared-workflows/pull/321/commits/9c4ac80c776a7d7aac9e20729d9d35a798692d8f)